src/styles/dropdown.css

```diff
@@ -1 +1,78 @@
 /* dropdown */
+
+.ui.selection.dropdown {
+  border-radius: 4px;
+  border: 1px solid var(--neutral-900);
+  padding: var(--spacing-1) var(--spacing-4) var(--spacing-1) var(--spacing-2);
+  min-width: 0;
+  min-height: 0;
+  font-family: var(--gotham);
+}
+
+.ui.selection.dropdown .divider.text {
+  color: var(--shade-black);
+  line-height: 1.5rem;
+}
+
+.ui.selection.dropdown .divider.default.text {
+  color: var(--neutral-900);
+}
+
+/** Focus **/
+.ui.selection.dropdown:not(.error):focus {
+  border-color: var(--primary-pms-362_900);
```

Comment:
None of our variables should use underscores. I think I missed that when the variables were first added. My bad. Do you mind updating them to use hyphens?

Reply:
Sure! Nice catch!